test/e2e_node/pod_conditions_test.go

```diff
@@ -134,6 +134,67 @@ func runPodFailingConditionsTest(f *framework.Framework, hasInitContainers, chec
 		// Verify PodReady is not set (since sandboxcreation is blocked)
 		_, err = getTransitionTimeForPodConditionWithStatus(p, v1.PodReady, false)
 		framework.ExpectNoError(err)
+
+		ginkgo.By("checking pod condition for a pod when volumes is created")
+
+		configmap := v1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "cm-that-unblock-pod-condition",
+			},
+			Data: map[string]string{
+				"key": "value",
+			},
+			BinaryData: map[string][]byte{
+				"binaryKey": []byte("value"),
+			},
+		}
+
+		_, err = f.ClientSet.CoreV1().ConfigMaps(f.Namespace.Name).Create(ctx, &configmap, metav1.CreateOptions{})
+		framework.ExpectNoError(err)
+
+		_, err = f.ClientSet.CoreV1().ConfigMaps(f.Namespace.Name).Get(ctx, "cm-that-unblock-pod-condition", metav1.GetOptions{})
+		framework.ExpectNoError(err)
```

Comment:
If the create succeed, we don't have to get it.

```suggestion
```

Reply:
cc @charles-chenzz

Reply:
@kannon92 paco was right, I get it here to check it explicitly during research and now we can confirm that it's created, we can remove it